addons/website/controllers/main.py

```diff
@@ -137,7 +137,12 @@ def country_infos(self, country, **kw):
 
     @http.route(['/robots.txt'], type='http', auth="public", website=True, sitemap=False)
     def robots(self, **kwargs):
-        return request.render('website.robots', {'url_root': request.httprequest.url_root}, mimetype='text/plain')
+        return request.make_response(f'''\
```

Comment:
> Why not keep the view ?

Seemed to be of limited worth given `robots_txt`.

> It will break some use case like custom view enabled (e.g. after a duplicate) to replace default sitemap

Is that really a thing?

> Or a module that disallow Semrush by default on all dbs.

?

> Remove it will make it impossible except with hack to alter the robots_text field... and request.httprequest.url_root no more available in your custo

What do you mean no more available? That you can't remove it anymore?

Reply:
> What do you mean no more available?

You don't have access to request.httprequest.url_root in robots_txt field.
So if you have a part of your website that is not on Odoo (e.g. you proxypass /xxx to another system) you cannot provide extra sitemap file in your robotx.txt

> Is that really a thing?
I suppose, since I use it myself ;)
On duplicate db for testing purpose, I enable a custom view to remove site map by a disallow all. But I can probably use a nginx config or other thing, but I just don't see the interest to remove the 'feature' that work as expected until now, to have something less customizable and no less risky... except if I miss a security hole here...

Reply:
OK fair enough.

> except if I miss a security hole here...

Nah it was just inconvenient.